packages/Confirmation/stories/Confirmation.examples.stories.js

```diff
@@ -0,0 +1,92 @@
+import React from "react";
+import { getStoryName } from "storybook/storyTree";
+import ExampleStory from "storybook/components/ExampleStory";
+import L10n from "@paprika/l10n";
+import ConfirmationExample from "./examples/ConfirmationExample";
+import ConfirmationExampleWithTrigger from "./examples/ConfirmationExampleWithTrigger";
+import ConfirmationExampleWithTriggerIcon from "./examples/ConfirmationExampleWithTriggerIcon";
+import ConfirmationExampleWithTriggerRaw from "./examples/ConfirmationExampleWithTriggerRaw";
+import ConfirmationExampleWithAsyncAction from "./examples/ConfirmationExampleWithAsyncAction";
+
+import Confirmation from "../src";
+
+const storyName = getStoryName("Confirmation");
+
+const confirmationStoryParameters = {
+  parameters: {
+    viewMode: "story",
+    options: {
+      isToolshown: false,
+      docs: { disable: true },
+    },
+  },
```

Comment:
I think you could use `exampleStoryParameters` instead from

`import { exampleStoryParameters } from "storybook/assets/storyParameters";`

Reply:
Unless `docs: { disable: true },` does something important?